web/src/main/java/no/nav/modiapersonoversikt/service/enhetligkodeverk/KodeverkConfig.kt

```diff
@@ -13,11 +13,31 @@ enum class KodeverkConfig(private val kilde: EnhetligKodeverk.Kilde) {
     DISKRESJONSKODER(FellesKodeverkKilde("Diskresjonskoder")),
     VALUTA(FellesKodeverkKilde("Valutaer")),
     TEMA(FellesKodeverkKilde("Tema")),
-    SF_TEMAGRUPPER(SfHenvendelseKodeverkKilde());
+    SF_TEMAGRUPPER(SfHenvendelseKodeverkKilde()),
+    NAVSKJEMAOGVEDLEGGSKODER(
+        FellesKodeverkFletter(
+            "NAVSkjemaOgVedleggskoder",
+            listOf(
+                FellesKodeverkKilde("NAVSkjema"),
+                FellesKodeverkKilde("Vedleggskoder")
+            )
+        )
+    );
 
     fun hentKodeverk(providers: KodeverkProviders) = kilde.hentKodeverk(providers)
 }
 
+class FellesKodeverkFletter(val nyttKodeverkNavn: String, val kodeverkKilder: List<FellesKodeverkKilde>) :
+    EnhetligKodeverk.Kilde {
+    override fun hentKodeverk(providers: KodeverkProviders): EnhetligKodeverk.Kodeverk {
+        val kodeverk = kodeverkKilder
+            .map { providers.fraFellesKodeverk(it.kodeverkNavn).kodeverk }
+            .reduce { a, b -> a + b }
+
+        return EnhetligKodeverk.Kodeverk(nyttKodeverkNavn, kodeverk)
```

Comment:
In hindsight ser jeg at vi sikkert kunne brukt noe a la dette i stedet, hvis vi er sikre på at det er distinkte nøkler
https://howtodoinjava.com/java/collections/hashmap/merge-two-hashmaps/